acceptance_tests/features/steps/unaddressed_steps.py

```diff
@@ -40,3 +45,19 @@ def _uac_callback(ch, method, _properties, body, context):
     context.expected_message_received = True
     ch.basic_ack(delivery_tag=method.delivery_tag)
     ch.stop_consuming()
+
+
+@when("the unaddressed batch is loaded, the print files are generated")
+def validate_unaddressed_print_file(context):
+    try:
+        subprocess.run(
+            ['docker', 'run',
+             '--env', f'RABBITMQ_SERVICE_HOST={Config.RABBITMQ_HOST}',
```

Comment:
I had to change this to:
```suggestion
             '--env', f'RABBITMQ_SERVICE_HOST=rabbitmq',
```

Reply:
Ah good point, it'll be using the internal Docker host/ports. I'll revert it back now.